examples/multi_ping.rs

```diff
@@ -13,20 +13,35 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         "39.156.69.79",
         "172.217.26.142",
         "240c::6666",
+        "2a02:930::ff76",
     ];
+    let ping_socket_v4 = Arc::new(PingSocket::new(socket2::Domain::IPV4)?);
```

Comment:
Maybe it can be
```rust
struct Pinger {
    v4_socket: AsyncSocket,
    v6_socket: AsyncSocket,
    ...
}
```

Reply:
Each raw socket may cause unwanted load because kernel replicate incoming packets to all existing raw socket, so I think we should maintain separate sockets for v4 and v6.